src/db/Procedures.ts

```diff
@@ -90,6 +108,52 @@ export default class Procedures {
     return result.rows[0].prosrc;
   }
 
+  public async triggerProcedure(
+    database: string,
+    procedure: string,
+    parameters: string[]
+  ) {
+    const client = await new Client({
+      host: this.address,
+      port: this.port,
+      password: 'asdf',
+      user: 'kpmg',
+      database,
+    });
+    client.connect();
+    const query = await client.query(
+      `CALL ${procedure}(${parameters.join(',')})`
+    );
+    client.end();
+    return query.rows[0];
+  }
+
+  public async getProcedureParameters(
+    database: string,
+    procedure: string
+  ): Promise<ProcedureParameter[]> {
+    const client = await new Client({
+      host: this.address,
+      port: this.port,
+      password: 'asdf',
+      user: 'kpmg',
```

Comment:
```suggestion
      password: this.password,
      user: this.user,
```
Can we do something like this? Would like to make it easier to pull these hard-coded values out later. Could make this change in all relevant functions here.

Reply:
These problems are resolved in #73

Reply:
Once #73 is finalized and merged into `feature`, I'll pull it into my branch and update this code to match the other functions.